web/src/app/webview/src/oskConfiguration.ts

```diff
@@ -46,34 +45,5 @@ export function buildEmbeddedGestureConfig(device: DeviceSpec) {
     }
   }
 
-  if(device.OS == DeviceSpec.OperatingSystem.Android) {
-    embeddedGestureConfig.createKeyTip = (vkbd) => {
```

Comment:
`createKeyTip` and `startLongpress` should be removed from `EmbeddedGestureConfig`.

I also think that the globe hint should be reimplemented in KeymanWeb, so we don't need any of this glue, and so it can be themed later. (This can be a separate pull)

Reply:
`createKeyTip` is used in `oskView` and `visualKeyboard`
`startLongpress` is used in `visualKeyboard`

@jahorton good with removing these?

----
globe hint split into separate issue #9596

Reply:
> `createKeyTip` and `startLongpress` should be removed from `EmbeddedGestureConfig`.

`EmbeddedGestureConfig` cleanup in 98660fef562ed2eed904e53b5fca568928a5f74f